src/window/glfw/Cargo.toml

```diff
@@ -28,6 +28,9 @@ documentation = "https://docs.rs/gfx_window_glfw"
 name = "gfx_window_glfw"
 
 [dependencies]
-glfw = "0.13"
+glfw = "0.14"
```

Comment:
I mean, we should bump the version of `gfx_window_glfw` to `0.15`, given that dependency change